cmd/exporter.go

```diff
@@ -80,25 +81,27 @@ func main() {
 
 	cgroup.SetSliceHandler()
 
-	if modelServerEndpoint != nil {
+	if *modelServerEndpoint != "" {
 		klog.Infof("Initializing the Model Server")
 		config.SetModelServerEndpoint(*modelServerEndpoint)
+		model.InitEstimateFunctions(collector_metric.ContainerMetricNames, collector_metric.NodeMetadataNames, collector_metric.NodeMetadataValues)
 	}
+
 	collector_metric.InitAvailableParamAndMetrics()
 
 	if *enableGPU {
-		klog.Infof("Initializing the GPU collector")
-		err := gpu.Init()
+		klog.Infof("Initializing the Accelerator collector")
```

Comment:
keep GPU. Accelerator is vague, not straightforward to end user debugging.

Reply:
@rootfs updated!